creation/web_base/condor_startup.sh

```diff
@@ -600,7 +608,7 @@ else
 fi
 
 
-((random100=$RANDOM%100))
+((random100=RANDOM%100))
```

Comment:
Is it intentional that $ has been removed? I noticed on line 91 of this file that the $ symbol is retained when using RANDOM